subgraphs/aave-forks/src/constants.ts

```diff
@@ -28,6 +28,7 @@ export namespace Network {
   export const XDAI = "XDAI"; // aka Gnosis Chain
 
   export const METIS = "andromeda";
+  export const BASE = "base";
```

Comment:
nit: although we use this inside equalsIgnoreCase() so it shouldn't matter, but we can use uppercases here for base and metis like for all other networks